### PR TITLE
endpoint for face recognizer implemented

### PR DESCRIPTION
**Pull request by GDayne**
endpoint for face recognizer implemented._
it works with a reference image(a file) and returns the percentaje of similarity

### CODE REVIEW

File: routes/recognizer_routes.py [hunk 1]
```diff
@@ -41,4 +44,40 @@ def recognize_object_from_zip():
     except Exception as e:
         return jsonify({"success": False, "message": "An error occurred", "error": str(e)}), 500
 
-#TODO nuevo endpoint devolver el json de classes_yolo (GET)
\ No newline at end of file
+@face_recognition_blueprint.route('/face_recognition', methods=['POST'])
```
Comment (Eynar92) [inline routes/recognizer_routes.py:47]: Todo: Rename to an appropriate one

File: utils/file_utils.py [hunk 2]
```diff
@@ -52,5 +52,12 @@ def download_file_from_url(url):
                 f.write(chunk)
     return file_name + extension
 
+def save_image(file):
+    filename = "imagen de referencia"
```
Comment (Eynar92) [inline utils/file_utils.py:56]: TODO: Use image name

File: routes/recognizer_routes.py [hunk 3]
```diff
@@ -41,4 +44,40 @@ def recognize_object_from_zip():
...
+    model_type = data.get('model_type')
+    confidence_threshold = (data.get('confidence_threshold', 0.1))
+    word = data.get('word')
+    image_file_reference = request.files.get('image_file_reference')
+    print("guardara la imagen")
+    print(image_file_reference)
+    reference_path = save_image(image_file_reference)
+    print("guardo la imagen")
+
+    try:
```
Comment (Joselian2204) [inline routes/recognizer_routes.py:63]: TODO: Try Except code could be replace with a method placed in utils to reduce the logic in the endpoint

**Comment by Eynar92** [comment]
Make the respective changes for the next PR

**Pull request merged**